src/additional/nisBR.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created for project jquery-validation.
+ * @Description Brazillian PIS or NIS number (Número de Identificação Social Pis ou Pasep) is the equivalent of a
+ * Brazilian tax registration number NIS of PIS numbers have 11 digits in total: 10 numbers followed by 1 check numbers
+ * that are being used for validation.
+ * @copyright (c) 25/10/2016 22:40, Cleiton da Silva Mendonça
```

Comment:
I think this is a copy-paste error. Can you please adjust the date to the right one.